src/test/java/com/google/api/codegen/testdata/discoveries/php/php_bigquery.v2.json.baseline

```diff
@@ -135,39 +119,34 @@ echo '<pre>', var_export($response, true), '</pre>', "\n";
  *    https://console.developers.google.com/apis/api/bigquery
  * 2. This sample uses Application Default Credentials for authentication.
  *    If not already done, install the gcloud CLI from
- *    https://cloud.google.com/sdk/ and run
+ *    https://cloud.google.com/sdk and run
  *    `gcloud beta auth application-default login`
+ *    For more information, see:
+ *    https://developers.google.com/identity/protocols/application-default-credentials
  * 3. Install the PHP client library with Composer. Check installation
  *    instructions at https://github.com/google/google-api-php-client.
  */
 
 // Autoload Composer.
 require_once __DIR__ . '/vendor/autoload.php';
 
-// Use Application Default Credentials for authentication when running locally.
-// For more information, see:
-// https://developers.google.com/identity/protocols/application-default-credentials
 $client = new Google_Client();
 $client->setApplicationName('Google-BigquerySample/0.1');
 $client->useApplicationDefaultCredentials();
 $client->addScope('https://www.googleapis.com/auth/cloud-platform');
 
 $service = new Google_Service_Bigquery($client);
 
-
-// TODO: Change placeholder below to desired parameter value for the `listDatasets` method:
-
 // Project ID of the datasets to be listed
-$projectId = '{MY-PROJECT-ID}';
+$projectId = '';  // TODO: Update placeholder value.
 
-
-$optParams = array();
+$optParams = [];
 do {
   $response = $service->datasets->listDatasets($projectId, $optParams);
 
-  foreach ($response['datasets'] as $dataset_list._datasets) {
-    // TODO: Change code below to process each `dataset_list._datasets` resource:
-    echo '<pre>', var_export($dataset_list._datasets, true), '</pre>', "\n";
+  foreach ($response['datasets'] as $datasets) {
```

Comment:
I believe so, `$response['datasets']` is an array of `Dataset`, so this should be correct.